src/elements/common/header/Header.js

```diff
@@ -33,10 +34,11 @@ const Header = ({ isHeaderLogoVisible = true, view, isSmall, searchQuery, onSear
             {isHeaderLogoVisible && <Logo isSmall={isSmall} url={logoUrl} />}
             <div className="be-search">
                 <input
-                    aria-label="search"
+                    aria-label={formatMessage(messages.searchPlaceholder)}
+                    data-testid="be-search-input"
                     disabled={!isFolder && !isSearch}
                     onChange={search}
-                    placeholder={intl.formatMessage(messages.searchPlaceholder)}
+                    placeholder={formatMessage(messages.searchPlaceholder)}
```

Comment:
Since it's being used in two places, I'd prefer we use a variable, but this is fine as well